<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="devSelector">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="protoSelector">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>all</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HTTP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HTTPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DNS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SMTP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TCP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ICMP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPv4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPv6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARP</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>状态提示：</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>协议选择：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>网卡选择：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="outputTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1231</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="protocolAnalyse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="rawData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="rawData_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tcpFlowOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>1231</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>协议解析:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>包数据:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>流跟踪：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopTrack">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>660</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>停止跟踪</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
